cwf/gateway/deploy/roles/ovs/tasks/debian.yml

```diff
@@ -44,7 +44,7 @@
        - libopenvswitch
        - openvswitch-common
        - openvswitch-switch
-       - python-openvswitch
+       - python3-openvswitch
```

Comment:
In line 40 above it states version "2.12.0", however, the logs in the integ tests indicate we are now using "2.15.4-9". "2.12.0" is the version of the _old_ `python-openvswitch`